components/clearbutton/stories/clearbutton.stories.js

```diff
@@ -55,3 +46,13 @@ export default {
 
 export const Default = Template.bind({});
 Default.args = {};
+
+export const OverBackground = Template.bind({});
+OverBackground.args = {
+	variant: "overBackground",
+	styles: {
+		"background-color": "rgb(15, 121, 125);", 
+		"color": "rgb(15, 121, 125);",
+		"padding": "15px 20px;"
+	}
```

Comment:
nice! this makes a lot of sense pulled out this way as opposed to how it was before